discovery-provider/src/tasks/index.py

```diff
@@ -277,18 +277,30 @@ def index_blocks(self, db, blocks_list):
             total_user_changes, user_ids = user_state_update(
                 self, update_task, session, user_factory_txs, block_number, block_timestamp)
             user_state_changed = total_user_changes > 0
+            logger.info(
```

Comment:
aren't these covered by the contract address logs above? what benefit does this give us vs using the existing logs?